crates/bevy_core_pipeline/src/fxaa/mod.rs

```diff
@@ -47,7 +47,7 @@ pub struct Fxaa {
 
     /// Use lower sensitivity for a sharper, faster, result.
     /// Use higher sensitivity for a slower, smoother, result.
-    /// Ultra and Turbo settings can result in significant smearing and loss of detail.
+    /// Ultra and Extreme settings can result in significant smearing and loss of detail.
```

Comment:
Perhaps these should link to their respective settings? That would help prevent such discrepancies in the future

Reply:
Fixed :+1: